src/App.jsx

```diff
@@ -127,6 +216,29 @@ function App() {
               <p className="amount">{formatCurrency(results.monthlyWithdrawal)}</p>
```

Comment:
⚖️ **Midlevel Developer**

Just noticed that we calculate total retirement income as the sum of withdrawal and social security regardless of when social security starts. Shouldn't the calculation reflect that social security doesn't kick in until `yearsUntilSocialSecurity`? Right now, the displayed "Total Monthly Retirement Income" combines them immediately, which might mislead users. Maybe add logic to show pre- and post-social security income separately?